src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -27,7 +32,7 @@ public Movie add(Movie movie) {
             if (transaction != null) {
                 transaction.rollback();
```

Comment:
The transaction rollback is correctly handled, but consider logging the exception before throwing a `DataProcessingException` to aid in debugging.